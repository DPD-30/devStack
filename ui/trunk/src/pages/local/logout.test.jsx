import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesLocalLogout from 'pages/local/logout.jsx'

jest.unmock('pages/local/logout.jsx')


describe('', () => {
    it('PagesLocalLogout should match snapshot', () => {
        const comp = renderToString(<PagesLocalLogout />)

        expect(comp).toMatchSnapshot()
    })
})
