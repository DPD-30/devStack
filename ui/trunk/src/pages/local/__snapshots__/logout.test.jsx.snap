// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesLocalLogout should match snapshot 1`] = `
<button class="btn btn-secondary"
        data-reactroot
>
  Logout
</button>
`;
